packages/web-components/src/components/ic-toast/ic-toast.stories.mdx

```diff
@@ -341,3 +341,90 @@ import readme from "./readme.md";
       </ic-toast-region>`}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  autoDismissTimeout: 5000,
+  dismissButtonAriaLabel: "dismiss",
+  dismissMode: "manual",
+  heading: "This is the heading",
+  message: "This is the message",
+  neutralIconAriaLabel: "",
+  variant: "neutral",
+  actionSlot: "action",
+  neutralIconSlot: "neutral-icon",
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    parameters={{ loki: { skip: true } }}
+    argTypes={{
+      dismissMode: {
+        options: ["manual", "automatic"],
+        control: {
+          type: "inline-radio",
+        },
+      },
+      variant: {
+        options: ["neutral", "info", "warning", "error", "success"],
```

Comment:
I think this needs an empty string option too as the variant prop is optional so is allowed to be undefined as well (like in the Default story)